tools/scripts/docs-website/create-docs-pr.js

```diff
@@ -173,6 +179,64 @@ async function readReleaseNoteFile (file) {
   })
 }
 
+/**
+ * Extracts the supported browser versions from the specified JSON file and creates a support string.
+ *
+ * @param {string} version The new version.
+ * @param {string} browsersFile The filename where the supported browsers JSON is stored.
+ */
+async function getBrowserTargetStatement (version, browsersFile) {
+  console.log('Retrieving supported browser targets from file: ', browsersFile)
+
+  const browserData = await readJsonFile(process.cwd() + '/' + browsersFile)
+
+  const min = {}
+  const max = {}
+
+  for (let browser in browserData) {
+    const versions = browserData[browser]
+    min[browser] = Infinity
+    max[browser] = -Infinity
+    for (let browserVersion of versions) {
+      const versionNumber = Number(browserVersion.version)
+      min[browser] = min[browser] > versionNumber ? versionNumber : min[browser]
+      max[browser] = max[browser] < versionNumber ? versionNumber : max[browser]
+    }
+  }
+
+  const ANDROID_CHROME_VERSION = 100 // SauceLabs only offers one Android Chrome version
+
+  return (
+    `Version ${version} of the Browser agent was built for and tested against these browsers and version ranges: ` +
+    `Chrome ${min.chrome}-${max.chrome}, Edge ${min.edge}-${max.edge}, Safari ${min.safari}-${max.safari}, Firefox ${min.firefox}-${max.firefox}; ` +
+    `and for mobile devices, Android Chrome ${ANDROID_CHROME_VERSION} and iOS Safari ${min.ios}-${max.ios}. ` +
+    'Instrumentation and specific features may be compatible with other browsers or versions.'
+  )
+}
+
+/**
+ * Reads the contents of a JSON file into a JavaScript object
+ *
+ * @param {string} File path to a JSON file.
+ * @returns a JavaScript object representing the file
+ */
+async function readJsonFile (filename) {
```

Comment:
We have `fs-extra` as a package which has a method for reading json files.

Reply:
You can change the require at the top `const fs = require('fs')` to `const fs = require('fs-extra')`. The package provides the same API surface as the built-in `fs` with additional functions.